docs/source/gravitysensors/tilt.rst

```diff
@@ -1,7 +1,76 @@
 .. include:: ../global.rst
 
-Tilt Sensor Setup
-====================
+Tilt Setup
+==========
 
-.. todo:: Write the Tilt Sensor Setup page
+To associate your a Tilt with Fermentrack, simply place the Tilt near the Raspberry Pi you are using at an angle (not vertical or horizontal) and it will automatically detect it (The Tilt enters a sleep mode when horizontal or vertical).
```

Comment:
I do not want to admit how many times I ran into this (the TIlt going to sleep and me not thinking about it).